pkg/proctree/nodes/process.go

```diff
@@ -0,0 +1,239 @@
+package nodes
+
+import "sync"
+
+// Arguments to feed or retrieve a process node
+
+type ProcessInfo struct {
+	Tid       int32
+	NsTid     int32
+	Pid       int32
+	NsPid     int32
+	StartTime uint64
+	ExitTime  uint64
+}
+
+type FileInfo struct {
+	Name      string
+	Path      string
```

Comment:
So this is the `fullpath` right? Do we get a `cmdline` as well (I guess not in the `FileInfo`)?